tools/cli/help.txt

```diff
@@ -500,14 +496,6 @@ Options:
                   the current versions, if required to satisfy all package version
                   constraints.
 
-
->>> logs
```

Comment:
We should also remove the associated code from `tools/cli/commands.js` (and wherever it branches out to!):

https://github.com/meteor/meteor/blob/9b056f5e911513fb3df716bd524d504cceb863f4/tools/cli/commands.js#L1431-L1444

Reply:
The reason I figured not to do that was because we don't plan on just deleting this outright, do we? I thought we had vague plans to fix it and maybe that code would be helpful. Then again, it'll still be in git history. The code just makes a call to the galaxy-server logs endpoint, so I was thinking we could leave it in, undocumented and unsupported, and eventually fix that endpoint once Logger2 is pushed out.

Reply:
Ehh I'm just gonna take it out – I think you're right. Plus the /logs endpoint doesn't even exist anymore

Reply:
Yeah, we can add it back easily!  I don't believe the `meteor logs` command has worked for two years now, so it's not a breaking change to remove it and put it back later.